example.php

```diff
@@ -2,7 +2,7 @@
 
 require __DIR__ . '/vendor/autoload.php';
 
-LightStep::initGlobalTracer('examples/trivial_process', '{your_access_token}');
+LightStep::initGlobalTracer('examples/trivial_process', '61c90a839a46c996e79c56afa1f116b8');
```

Comment:
Let's make sure we revoke this and replace it with the default text

Reply:
nice catch, i already revoked